src/proteus/dswx_hls.py

```diff
@@ -3943,14 +3953,17 @@ def _populate_dswx_metadata_processing_parameters(
         dswx_metadata_dict['FOREST_MASK_LANDCOVER_CLASSES'] = \
             ','.join([str(c) for c in forest_mask_landcover_classes])
     else:
-        dswx_metadata_dict['FOREST_MASK_LANDCOVER_CLASSES'] = '-'
+        dswx_metadata_dict['FOREST_MASK_LANDCOVER_CLASSES'] = 'EMPTY'
+
+    # ocean masking
+    dswx_metadata_dict['OCEAN_MASKING_ENABLED'] = \
+        'TRUE' if apply_ocean_masking else 'FALSE'
```

Comment:
Please check my understanding: If a shoreline shapefile is not provided, then ocean masking will not be applied, regardless of how `apply_ocean_masking` is set.

If so, then in that scenario, should `OCEAN_MASKING_ENABLED` also be set to `False`? Currently, it looks like the code will set it to `TRUE`?

Reply:
This is verified in `_check_ancillary_inputs()` if `apply_ocean_masking` is set but a shoreline shapefile is not provided, the SAS will raise an error in that function (btw, I'm can simplify the next condition `if apply_ocean_masking and shoreline_shapefile:`).